### DISCUSSION TITLE
v0.15.0 requires some data work, volunteers needed!

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#989] [Announcements]
# Description

Since the start there's been a bug that incorrectly formats the json string in the `data` column in the `results` table. This needs to be fixed prior to moving on to backup/restore and import/export work.

The bulk of this work will be occurring in https://github.com/alexjustesen/speedtest-tracker/pull/812 and will provide a couple of solutions to migrate and fix the data.

## How do you participate?

Either subscribe to notifications on this discussion topic or post a comment below. I'll be updating this discussion with beta releases as they come out.

**Comment by alexjustesen** [comment]
I need to do some performance testing, can someone give me a ballpark number of results their databases have? My current testing scenario is build off 1 test an hour for the last 365 days, figuring this is the high end of the usage.

**Comment by masterwishx** [comment]
i have SQLiteDB tests IDs (13937) from Dec, 2022 until now, the test is every 10 min

**Reply by masterwishx** [reply]
a some few first months the test was every hour ,when used with old speedtest-tracker ,then when migrated to this fully ,changed to every 10 min

**Reply by masterwishx** [reply]
Checked , first 10 months was 1 hour , then every 10 min also now

**Reply by alexjustesen** [reply]
Holy crap man that's frequent. 

If I run into a bottleneck anywhere I'm guess it's going to be sqlite.

**Reply by masterwishx** [reply]
Yes, I know, I had problems with provider speed in docsis 3.0, so was need to set every 10min, and some time it's back  , so leaved as is.
I have also mariadb, postgresql, just not using for speedtest-tracker...
